javascripts/discourse/components/modal/dev-toolbox-modal.gjs

```diff
@@ -0,0 +1,219 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import DButton from "discourse/components/d-button";
+import DModal from "discourse/components/d-modal";
+import DiscourseURL from "discourse/lib/url";
+import dIcon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+import ComboBox from "select-kit/components/combo-box";
+
+export default class DevToolboxModal extends Component {
+  @tracked commonSetting;
+  commonSettings = [
```

Comment:
if it's constant and doesn't use any properties then it's best to extract it to a const.